shared/js/initApp.js

```diff
@@ -21,6 +21,11 @@ var baseOptions = {
     if (appOptions.level.isProjectLevel) {
       return;
     }
+    if (appOptions.channel) {
+      // Don't send the levelSource to Dashboard for channel-backed levels.
+      // (The levelSource is already stored in the channels API.)
+      delete report.program;
```

Comment:
are there other things that we should be deleting, like the image? (LevelSourceImages can't be saved without a LevelSource)